src/hardware/index.html

```diff
@@ -10,19 +10,19 @@
     <a href="/hardware/yellow-v1.2-standard">
       <div class="layout vertical center">
         <img src="/images/yellow-power-supply.png" />
-        <h3>Standard Rev 1.2</h3>
+        <h3>Standard power supply, Rev 1.2</h3>
```

Comment:
To me, this is more confusing: The PoE PCB also supports a "standard power supply" (the barrel plug). This make it sound like the other PCB does not support a standard power supply... :thinking: